packages/lib/src/index.ts

```diff
@@ -115,6 +115,8 @@ export { scaleGamma } from './vis/scaleGamma';
 export { useCanvasEvents } from './interactions/hooks';
 export { default as Box } from './interactions/box';
 
+export { COLOR_SCALE_TYPES, AXIS_SCALE_TYPES } from '@h5web/shared';
```

Comment:
Sorry for dumping this in here. I'm exporting these arrays and types (`ColorScaleType` and `AxisScaleType`) so I can use them in Braggy.